tests/multi_step/test_correctness_llm.py

```diff
@@ -57,7 +57,7 @@ def test_multi_step_llm(
                            GPU -> CPU output transfer
       num_prompts: number of example prompts under test
       num_logprobs: corresponds to the `logprobs` argument to the OpenAI
-                    completions endpoint; `None` -> no logprobs
+                    completions endpoint; `None` -> no logprobs 
```

Comment:
Miss added?

Reply:
Thank for this comment. It appears there is no difference between the line you are highlighting vs the corrected version. However, I assume that you are pointing out that the comment incorrectly says that `None` -> no logprobs when it should say that `None` means that 1 logprob is returned? Anyway I made that change.